packages/compass-aggregations/src/components/stage-wizard/index.tsx

```diff
@@ -56,11 +74,12 @@ export const StageWizard = ({ id, onCancel, onApply }: StageWizardProps) => {
         <Body weight="medium">{useCase.title}</Body>
         <Badge>{useCase.stageOperator}</Badge>
       </div>
-      <useCase.wizardComponent />
+      <useCase.wizardComponent onChange={onChange} />
       <div className={cardFooterStyles}>
+        {syntaxError && <WarningSummary warnings={[syntaxError.message]} />}
         <div className={cardActionStyles}>
           <Button onClick={onCancel}>Cancel</Button>
-          <Button onClick={onApply} variant="primary">
+          <Button onClick={onApply} variant="primary" disabled={!!syntaxError}>
```

Comment:
```suggestion
          <Button onClick={onApply} variant="primary" disabled={!!syntaxError || !value}>
```
Maybe we disable it for empty value as well?